### PR TITLE
bugfix KubernetesApiResponse cannot be cast to class KubernetesObject

### PR DESCRIPTION
**Pull request by lin1005q**

### CODE REVIEW

File: extended/src/main/java/io/kubernetes/client/extended/kubectl/KubectlPatch.java [hunk 1]
```diff
@@ -44,10 +44,22 @@ public ApiType execute() throws KubectlException {
...
+                .throwsApiException()
+                .getObject();
+      } else {
+        return (ApiType)
+            genericKubernetesApi
+                .patch(name, patchType, patchContent)
+                .throwsApiException()
+                .getObject();
+      }
+    } catch (Exception e) {
```
Comment (brendandburns) [inline extended/src/main/java/io/kubernetes/client/extended/kubectl/KubectlPatch.java:61]: Please catch a specific exception, not all exceptions.

Comment (lin1005q) [reply extended/src/main/java/io/kubernetes/client/extended/kubectl/KubectlPatch.java]: please review again

**Review by yue9944882 — COMMENTED** [review]
/lgtm
/approve

@lin1005q are you expecting cherry-picking this to any history release?

**Comment by lin1005q** [comment]
> /lgtm
> /approve
> 
> @lin1005q are you expecting cherry-picking this to any history release?

I haven't done it before, but I want to give it a try.
I saw the KubectlPatch class began to appear on the release-10 branch. Please confirm which branch I need to cherry-picking on?

**Pull request merged**